### DISCUSSION TITLE
Bring to Jeedom

### DISCUSSION THREAD

**Discussion by yozart** [krahabb/meross_lan#169] [Ideas]
Hello Krahabb,

Did you ever thought about bringing this into Jeedom. I guess porting it wouldn't be so complicated. The Jeedom community is really looking forward to such plugin even if we should pay a few bucks for it ! Current plugins available for meross are buggy or not managed any more by the developpers.

Would be great to have a quality and cloudless plugin as yours !

**Comment by krahabb** [comment]
I took a look at the Jeedom solution and it looks intriguing but by the look of it the plugin is written in php and that would be another learning curve for me..
At the moment that's really unpractical but maybe someone else is up for the challenge!

Thank you for reporting this anyway: this is something I was totally unaware of and could maybe prove useful in the future

**Reply by yozart** [reply]
Indeed you can create plugins in PHP but also in Python ! some additionnal info here :
https://doc.jeedom.com/en_US/dev/daemon_plugin

Maybe you will understand what is required to get this ported ? Or if you know anyone who can do it !

**Reply by krahabb** [reply]
Hello yozart, sorry to be this late, even if I recognize it would be nice and useful to have this kind of port I really can't afford to make it (I'm still on a learning curve in both python and HA api) but if someone else is up for the challenge I would definitely support that.
I mean I could help explaining meross_lan implementation choices and behaviours in order to help/ease the porting process.
